mysql/plan.sh

```diff
@@ -52,7 +52,7 @@ do_build() {
           -DCURSES_INCLUDE_PATH="$(pkg_path_for core/ncurses)/include" \
           -DCURSES_LIBRARY="$(pkg_path_for core/ncurses)/lib/libcurses.so" \
           -DWITH_SSL=yes \
-          -DOPENSSL_INCLUDE_DIR="$(hab pkg path core/openssl)/include" \
+          -DOPENSSL_INCLUDE_DIR="$(pkg_path_for core/openssl)/include" \
```

Comment:
with `pkg_path_for` (but not `hab pkg path`), you can just say `openssl`, not `core/openssl`, and it will work, FYI. No need to change it here, but just so you know. Works nice if you have to change origins in your deps.